src/APILookupPango.txt

```diff
@@ -59,6 +59,15 @@ extend: Attribute
 struct: AttrFontFeatures
 noCode: true
 
+struct: Font
+noCode: get_features
+noCode: get_hb_font
+# XXX We could probably remove this if we shipped proper harfbuzz bindings
+addAliases: start
+	alias hb_feature_t = void;
+	alias hb_font_t = void;
+addAliases: stop
```

Comment:
It would be better to put this in a version block: `version 1.44: start` and `version: end`.

This so you can still regenerate the code when you have a older version of pango installed.